### PR TITLE
Update cs0071.md

### PR DESCRIPTION
**Pull request by Youssef1313**
- Removed extra spaces.
- Fixed typo: `Itest` => `ITest`.

### CODE REVIEW

File: docs/csharp/language-reference/compiler-messages/cs0071.md [hunk 1]
```diff
@@ -1,49 +1,51 @@
...
+helpviewer_keywords:
   - "CS0071"
 ms.assetid: 787cbeae-fb2b-455a-ba10-811b956ed170
 ---
 # Compiler Error CS0071
-An explicit interface implementation of an event must use event accessor syntax  
+
+An explicit interface implementation of an event must use event accessor syntax
+
+ When explicitly implementing an [event](../keywords/event.md) that was declared in an interface, you must use manually provide the `add` and `remove` event accessors that are typically provided by the compiler. The accessor code can connect the interface event to another event in your class (shown later in this topic), or to its own delegate type. For more information, see [How to: Implement Interface Events](../../programming-guide/events/how-to-implement-interface-events.md).
```
Comment (rpetrusha) [inline docs/csharp/language-reference/compiler-messages/cs0071.md]:
```suggestion
 When explicitly implementing an [event](../keywords/event.md) that was declared in an interface, you must manually provide the `add` and `remove` event accessors that are typically provided by the compiler. The accessor code can connect the interface event to another event in your class (shown later in this topic) or to its own delegate type. For more information, see [How to: Implement Interface Events](../../programming-guide/events/how-to-implement-interface-events.md).
```

File: docs/csharp/language-reference/compiler-messages/cs0071.md [hunk 2]
```diff
@@ -1,49 +1,51 @@
...
+interface ITest
+{
+    event MyEvent Clicked;
+}
+
+class Test : ITest
+{
+    event MyEvent ITest.Clicked;  // CS0071
+
+    // Try the following code instead.
```
Comment (rpetrusha) [inline docs/csharp/language-reference/compiler-messages/cs0071.md]:
```suggestion
// Try the following code instead.
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for submitting this PR, @Youssef1313. I've suggested some changes.

File: docs/csharp/language-reference/compiler-messages/cs0071.md (hunk 2 above)
Comment (Youssef1313) [reply docs/csharp/language-reference/compiler-messages/cs0071.md]: This line indentation seems correct to me, there is another line that has incorrect indentation. Will fix it now.

**Comment by rpetrusha** [comment]
Thanks, @Youssef1313. I'll merge your PR now.

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**